packages/server/modules/core/tests/integration/email-verification.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { describe } from 'mocha'
+import { updateUserEmailFactory } from '@/modules/core/repositories/userEmails'
+import knexInstance from '@/db/knex'
+import { createUser } from '@/modules/core/services/users'
+import {
+  createRandomEmail,
+  createRandomPassword
+} from '@/modules/core/helpers/testHelpers'
+import { markUserEmailAsVerifiedFactory } from '@/modules/core/services/users/email-verification'
+import { expect } from 'chai'
+import { USER_EMAILS_TABLE_NAME } from '@/modules/core/dbSchema'
+
+const userEmailTable = knexInstance(USER_EMAILS_TABLE_NAME)
+
+describe('Verification @user-emails', () => {
+  it('should mark user email as verified', async () => {
```

Comment:
Do we really need the test files to be this granular? There is an overheard to each test file - initializing a new test context, running `beforeEachContext` etc.

Some if not all of these can probably be merged into the same test file